regression_runner/requirements.txt

```diff
@@ -0,0 +1,19 @@
+aiohttp==3.6.2
+async-timeout==3.0.1
+attrs==19.3.0
+certifi==2019.11.28
+chardet==3.0.4
+click==7.1.1
+coloredlogs==14.0
+future==0.18.2
+humanfriendly==8.1
+idna==2.9
+multidict==4.7.5
```

Comment:
I had to add
`wheel==0.34.2`
before PyYAML to make PyYAML install successfuly.
Otherwise it failed with

> Building wheels for collected packages: PyYAML
>   Running setup.py bdist_wheel for PyYAML ... error
>   Complete output from command /home/frosch/daten/ottd/bananas_api/.env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-loj4h5vd/PyYAML/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/pip-wheel-cufe2hxu --python-tag cp37:
>   usage: -c [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
>      or: -c --help [cmd1 cmd2 ...]
>      or: -c --help-commands
>      or: -c cmd --help
>   
>   error: invalid command 'bdist_wheel'
>   
>   ----------------------------------------
>   Failed building wheel for PyYAML
>   Running setup.py clean for PyYAML
> Failed to build PyYAML

Reply:
This is most likely a local issue. Possibly because your pip is a bit old. Docker is building this repository fine, without the need for this package.

Wheel is only needed if you are creating wheels, which you are not. Or should not be doing anyway.

Possibly some global pip settings that is set?

Anyway, not a problem I can fix in this repository :(